src/routes/Gavin/modules/GeneNetworkScore.js

```diff
@@ -0,0 +1,53 @@
+// ------------------------------------
+// Constants
+// ------------------------------------
+export const SET_GN_SCORES = 'Gavin.SET_GN_SCORES'
+
+export const constants = { SET_GN_SCORES }
+
+// ------------------------------------
+// Action creators
+// ------------------------------------
+export function setGeneNetworkScores (scores) {
+  return {
+    type    : SET_GN_SCORES,
```

Comment:
Here too, you load the scores and the event happens when the scores arrive in the browser. So SCORES_LOADED or so.